### PR TITLE
feat: add --no-husky flag

### LINKED ISSUE #101: feature: remove husky
### Current Behavior

when init project with dts-cli, it installs husky, which breaks the hooks of git when in monorepo.

### Desired Behavior

no husky installed, or there is options in the cli

### Suggested Solution

add a question to decide if husky needed.

### PR DESCRIPTION
**Pull request by ToppleTheNun**
add "--no-husky" flag for disabling husky
inclusion in generated projects.

closes #101

### CODE REVIEW

**Review by aladdin-add — COMMENTED** [review]
LGTM, can you also update the readme?

File: src/templates/basic.ts [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { Template } from './template';
 const basicTemplate: Template = {
   name: 'basic',
   dependencies: [
-    'husky',
```
Comment (aladdin-add) [inline src/templates/basic.ts:6]: what do you think keep the `husky` here but removing it when `opts.husky` is `false`? I'm asking it as it's the default.

Comment (ToppleTheNun) [reply src/templates/basic.ts:6]: That makes more sense given how it's an opt-out rather than an opt-in. Will work on that.

Comment (ToppleTheNun) [reply src/templates/basic.ts]: Done.

**Review by aladdin-add — APPROVED** [review]
LGTM, thanks! 💯

**Pull request merged**